packages/application-shell/src/configure-apollo.js

```diff
@@ -66,6 +66,8 @@ const fragmentMatcher = new IntrospectionFragmentMatcher({
 
 const typeNamesWithoutIdAsIdentifier = ['Project', 'BaseMenu', 'NavbarMenu'];
 
+const typeNameWithReferencedEntity = ['Reference', 'DeliveryItem'];
```

Comment:
I think this is named accordingly to what this points to 😋

Reply:
Thanks for the extraction.